packages/components/src/core/AutocompleteMultiColumn/style.ts

```diff
@@ -0,0 +1,171 @@
+import { Paper, Popper } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import { ReactElement } from "react";
+import InputSearch from "../InputSearch";
+import {
+  CommonThemeProps,
+  fontCapsXxxxs,
+  getBorders,
+  getColors,
+  getCorners,
+  getShadows,
+  getSpaces,
+  getTypography,
+} from "../styles";
+
+export interface StyleProps extends CommonThemeProps {
+  count?: number;
+  icon?: ReactElement;
+  search?: boolean;
+  title?: string;
+  columnWidth?: number;
+}
+
+const doNotForwardProps = [
+  "anchorEl",
+  "count",
+  "keepSearchOnSelect",
+  "search",
+  "InputBaseProps",
+  "title",
+  "PopperBaseProps",
+  "onClickAway",
+  "ClickAwayListenerProps",
+];
+
+export const StyledPopper = styled(Popper, {
+  shouldForwardProp: (prop: string) =>
+    !doNotForwardProps.includes(prop) || prop === "anchorEl",
+})`
+  ${(props: StyleProps) => {
+    const borders = getBorders(props);
+    const corners = getCorners(props);
+    const shadows = getShadows(props);
+    const spacings = getSpaces(props);
+
+    return `
+      background-color: white;
+      border: ${borders?.gray[100]};
+      border-radius: ${corners?.m}px;
+      box-shadow: ${shadows?.m};
+      padding-top: ${spacings?.l}px;
+      padding-bottom: 0;
+      padding-right: calc(${spacings?.l}px - ${spacings?.m}px);
+      padding-left: ${spacings?.l}px;
+      box-sizing: border-box;
+      z-index: 1400;
+    `;
+  }}
+`;
+
+export const StyledPaper = styled(Paper, {
+  shouldForwardProp: (prop: string) => !doNotForwardProps.includes(prop),
+})`
+  box-shadow: none;
+  margin: 0;
+  border-radius: 0;
+  padding-top: 0;
+  padding-bottom: 0;
+`;
+
+export const StyledAutocompletePopper = styled(Popper)`
+  ${(props: StyleProps) => {
+    const spacings = getSpaces(props);
+
+    return `
+      position: relative !important;
+      transform: none !important;
+      width: 100% !important;
+      box-shadow: none;
+      padding: 0;
+      border: none;
+    
+      && .MuiPaper-root .MuiAutocomplete-listbox {
+        padding: 0 ${spacings?.m}px ${spacings?.l}px 0;
+      }
+    `;
+  }}
+`;
+
+export const StyledAutocomplesWrapper = styled("div")`
```

Comment:
typo 😆 

s/StyledAutocomplesWrapper/StyledAutocompletesWrapper

Do you have https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker installed 👌 ?

Reply:
Ah, thanks for catching that. It's a useful extension. I just installed it :D